ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -152,8 +152,12 @@ internal class PaywallViewModelImpl(
             Logger.e("Activity is null, not initiating package purchase")
             return
         }
+        if (verifyNoActionInProgressOrStartAction()) {
+            return
+        }
```

Comment:
I thought about moving these inside the internal method, but no sense to perform extra operations without need IMO.